CredentialProvider.Microsoft/Resources.resx

```diff
@@ -456,4 +456,7 @@ NuGet workarounds
   <data name="ForcingCanShowDialogFromTo" xml:space="preserve">
     <value>Forcing canShowDialog from {0} to {1} due to setting from environment variable</value>
   </data>
+  <data name="UIFlowStarted" xml:space="preserve">
+    <value>Using the {0}  flow. User sign-in required in pop-up authentication window.</value>

```

Comment:
nit: User sign-in required in _a_ pop-up authentication window.

Reply:
yikes, nice catch. Fixing it now